recipes/imgui-sfml/all/patches/0001-conan-libs-2.5.patch

```diff
@@ -0,0 +1,20 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index fad29a6..9a43dcf 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -93,12 +93,12 @@ add_library(ImGui-SFML::ImGui-SFML ALIAS ImGui-SFML)
+ 
+ target_link_libraries(ImGui-SFML
+   PUBLIC
+-    sfml-graphics
+-    sfml-system
+-    sfml-window
++	  ${CONAN_LIBS}
```

Comment:
Did you try compiling it without this patch?

The SFML recipe (and tons of other recipes) got improved in recent times. It might now produce the original upstream CMake targets required

Reply:
Nevermind, lets have a look at this somewhen else